### DISCUSSION TITLE
Timeframe for getStaticProps support in _app.js

### DISCUSSION THREAD

**Discussion by caribou-code** [vercel/next.js#15698] [Help] [answered]
As per [this previously answered question](https://github.com/vercel/next.js/discussions/10949#discussioncomment-43898) regarding _app.js not supporting getStaticProps, the answer says:

> This is just an initial restriction as we want to make sure we provide the right abstraction for it in _app

This will be a game changing addition for me, so I'm really excited to see it. Is there a known timeframe for this work?

**Comment by timneutkens** [comment, marked as answer]
Replied here: https://github.com/vercel/next.js/discussions/10949#discussioncomment-44898
